app/Http/Controllers/CompetitionsController.php

```diff
@@ -206,4 +209,40 @@ public function leaderboard(Competition $competition)
 
         return view('competitions.leaderboard', compact('competition', 'leaderboard', 'pending', 'flagged'));
     }
+
+    /**
+     * Get the featured reportback form.
+     *
+     * @param  \Gladiator\Models\Competition  $competition
+     * @return \Illuminate\Http\Response
+     */
+    public function featuredReportbackForm(Competition $competition, Message $message)
+    {
+        $reportback = FeaturedReportback::where('competition_id', '=', $competition->id)->where('message_id', '=', $message->id)->first();
+
+        return view('competitions.featured_reportback', compact('competition', 'message', 'reportback'));
```

Comment:
If these are forms in a separate view, then continue to follow the conventions ideally. so `competitions.featured_reportback.edit`